src/Project/Command.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class Command extends Array<string> {
+  readonly cmd: string;
+  options?: string[];
+
+  constructor(cmd: string, options?: string[]) {
+    super();
+    this.cmd = cmd;
+    this.options = options;
+  }
+
+  strs(): string[] {
+    let ret: string[] = [this.cmd];
+    if (this.options !== undefined) {
+      ret.concat(this.options);
```

Comment:
```suggestion
      ret = ret.concat(this.options);
```

The `concat` returns new array but we didn't update the `ret` variable.

Reply:
This is fixed by hot-fix command class.